### ISSUE TITLE
amp-pinterest module fails lighthouse tests 100% of the time

### ISSUE THREAD

**Issue by ayumihamsaki** [ampproject/amphtml#19281]
The basic issue is that this module does not give an Accessibility Link Name, see screenshot:

![untitled](https://user-images.githubusercontent.com/17784082/48426919-9f943600-e75f-11e8-979e-0600bdc826b5.png)

Is it possible for the authors to add the following code:

```
role="link"
aria-label="Send Image to Pinterest"
```

Then I believe this should make this module pass lighthouse and give a better Accessibility Experience to people.

Also would it not be a good idea for all amp-modules to be **_required_** to pass basic lighthouse tests?

**Comment by aghassemi** [comment]
The anchor is fine (it is a `a` with proper text that say save. The big problem is the `Save` button on top of a `pin` which is a `span` without `role=button, `title` and `tabIndex` :(

**Comment by aghassemi** [comment]
![Screen Shot 2019-06-18 at 2 45 52 PM](https://user-images.githubusercontent.com/2099009/59722154-da6bba00-91d7-11e9-8ed2-7e616f07af56.png)

**Comment by ayumihamsaki** [comment]
Thanks for the extra info and making it a P1 label now. Will watch this issue and when it is merged I will re-run a lighthouse test and see how it goes.
